build.gradle

```diff
@@ -532,6 +532,10 @@ subprojects {
     testFixturesImplementation 'org.assertj:assertj-core'
   }
 
+  if (project.name != 'unsigned' && project.name != 'errorprone-checks') {
+    dependencies.implementation(project(':infrastructure:unsigned'))
+  }
```

Comment:
We make UInt64 available everywhere by default.  Have to exclude the unsigned module itself and errorprone-checks from that to avoid recursive dependency loops.